exercises/leap/test/leap_test.dart

```diff
@@ -24,5 +24,23 @@ void main() {
       final bool result = leap.leapYear(2000);
       expect(result, equals(true));
     }, skip: true);
+
+    test("is introduced every 4 years to adjust about a day before 400 A.D.", () {
+      final bool result = leap.leapYear(4);
+
+      expect(result, equals(true));
+    }, skip: true);
+
+    test("is skipped every 100 years to remove an extra day before 400 A.D.", () {
+      final bool result = leap.leapYear(300);
+
+      expect(result, equals(false));
+    }, skip: true);
+
+    test("is reintroduced every 400 years to adjust another day before 400 A.D.", () {
+      final bool result = leap.leapYear(400);
```

Comment:
@jvarness Not sure I understand this, how is the year "400" "before 400 A.D."?

Reply:
It's not lol, I'll fix this.